src/univariate/continuous/normalcanon.jl

```diff
@@ -1,24 +1,28 @@
 ## Canonical Form of Normal distribution
 
-immutable NormalCanon <: ContinuousUnivariateDistribution
-    η::Float64       # σ^(-2) * μ
-    λ::Float64    # σ^(-2)
-    μ::Float64       # μ
+immutable NormalCanon{T<:Real, S<:Real} <: ContinuousUnivariateDistribution
```

Comment:
Agreed. I had something ugly I was trying to avoid, but I can't recall now what that was. Easy enough to fix, I think.